RadarSDK/Include/RadarAddress.h

```diff
@@ -22,6 +22,22 @@ typedef NS_ENUM(NSInteger, RadarAddressConfidence) {
     RadarAddressConfidenceFallback NS_SWIFT_NAME(fallback) = 3
 };
 
+// Verification status enum for RadarAddress with values 'V', 'P', 'A', 'R', and 'U'
+typedef NS_ENUM(NSInteger, RadarAddressVerificationStatus) {
+    /// Unknown
+    RadarAddressVerificationStatusNone NS_SWIFT_NAME(none) = 0,
+    /// Verified: complete match was made between the input data and a single record from the available reference data
+    RadarAddressVerificationStatusVerified NS_SWIFT_NAME(verified) = 1,
+    /// Partially verified: a partial match was made between the input data and a single record from the available reference data
+    RadarAddressVerificationStatusPartiallyVerified NS_SWIFT_NAME(partiallyVerified) = 2,
+    /// Ambiguous: more than one close reference data match
+    RadarAddressVerificationStatusAmbiguous NS_SWIFT_NAME(ambiguous) = 3,
+    /// Reverted: record could not be verified to the specified minimum acceptable level. The output fields will contain the input data
+    RadarAddressVerificationStatusReverted NS_SWIFT_NAME(reverted) = 4,
+    /// Unverified: unable to verify. The output fields will contain the input data
+    RadarAddressVerificationStatusUnverified NS_SWIFT_NAME(unverified) = 5
+};
```

Comment:
could have done this just as a string, but since we're basing this type off of an external spec, seemed appropriate to turn into an `enum` client side as well.

Seems that we use numerical enums across the codebase, so I opted that way as well.